docs/ddl-of-carbondata.md

```diff
@@ -638,7 +638,12 @@ CarbonData DDL statements are documented here,which includes:
 
 
 ## CREATE DATABASE 
-  This function creates a new database. By default the database is created in Carbon store location, but you can also specify custom location.
+  This function creates a new database. By default the database is created in location 'spark.sql.warehouse.dir', but you can also specify custom location by configure 'spark.sql.warehouse.dir', the configuration 'carbon.storelocation' has been deprecated.
+
+  **Note:**
+    For simplicity, we recommended you remove the configuration of carbon.storelocation. If carbon.storelocaiton and spark.sql.warehouse.dir are configured to be different path, exception will be thrown when CREATE DATABASE and DROP DATABASE to avoid inconsistent database location.
```

Comment:
```suggestion
    For simplicity, we recommended you remove the configuration of carbon.storelocation. If carbon.storelocaiton and spark.sql.warehouse.dir are configured to different paths, exception will be thrown when CREATE DATABASE and DROP DATABASE to avoid inconsistent database location.
```

Reply:
I have modified code according to your suggestion